store/conf/attrs/zimbra-attrs.xml

```diff
@@ -4297,14 +4297,14 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
   <desc>Maximum percent of conflicting instances allowed before declining schedule request for a recurring appointment; default 0 (means decline on any conflict)</desc>
 </attr>
 
-<attr id="810" name="zimbraDistributionListSendShareMessageToNewMembers" type="boolean" cardinality="single" optionalIn="distributionList" since="6.0.0_BETA1">
+<attr id="810" name="zimbraDistributionListSendShareMessageToNewMembers" type="boolean" cardinality="single" optionalIn="distributionList,group" since="6.0.0_BETA1">
```

Comment:
these changes will go in 9.1.0 only and can not be backported to 9.0.0 or 8.8.15, I hope the expectations are already cleared with product management

Reply:
Yes Saket!